release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -1802,11 +1810,17 @@ See link:https://access.redhat.com/articles/6955985[Navigating Kubernetes API de
 [id="ocp-release-note-installer-bug-fixes_{context}"]
 ==== Installer
 
+* Previously, when you attempted to set the policy for a {gcp-first} service account, the API reported a `400: Bad Request` validation error. When you create a service account, it might take up to 60 seconds for the account to become active, and this causes the validation error. If this error occurs, create a service account with a true exponential backoff that lasts at least 60 seconds. (link:https://issues.redhat.com/browse/OCPBUGS-48187[*OCPBUGS-48187*])
```

Comment:
This has a workaround rather than a fix - seems like a known issue, not a bug fix